store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -56,6 +56,12 @@ public class VolumeConfigUtil {
      */
     private static final String DEFAULT_EXTERNAL_STORE_MANAGER = "com.zimbra.storemanagers.store.GenericStoreManager";
 
+    // standard regex for validation of an url
+    public static String URL_REGEX = "^https?:\\/\\/(?:www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b(?:[-a-zA-Z0-9()@:%_\\+.~#?&\\/=]*)$";
+
+    // name must start with a letter. It can contain word characters, dashes and dots
+    public static String NAME_REGEX = "^[a-zA-Z0-9][\\w-.]{1,62}$";
```

Comment:
Dots should not be allowed, also please make you have tested this.

Reply:
sure Rakesh

Reply:
Tested & removed dot validation